velox/type/tests/VariantTest.cpp

```diff
@@ -119,6 +139,7 @@ TEST(VariantTest, shortDecimalNull) {
   EXPECT_EQ(null.toJson(), "null");
   EXPECT_EQ(*null.inferType(), *DECIMAL(10, 5));
   EXPECT_THROW(variant::null(TypeKind::SHORT_DECIMAL), VeloxException);
+  EXPECT_THROW(null.value<TypeKind::SHORT_DECIMAL>().hash(), VeloxException);
```

Comment:
looks like other types do support null values for hash.

Reply:
What do you suggest the expected behavior to be?